### DISCUSSION TITLE
How to ignore .tsx

### DISCUSSION THREAD

**Discussion by Rykus0** [storybookjs/storybook#16145] [Help] [answered]
I'm currently getting an error that seems to be due to the tsx syntax in a React component, but it's using the vue loader (I have both react and vue components in my design system).

If I could tell the vue loader to ignore tsx files, I should be all set.  

Here is the error if it helps

Unexpected error while loading ./04-templates/sbl/ContactInfo.stories.tsx: Error: Module parse failed: Unexpected token (18:37)
File was processed with these loaders:
 * ./node_modules/@storybook/vue/node_modules/ts-loader/index.js
 * ./node_modules/@storybook/source-loader/dist/cjs/index.js
You may need an additional loader to handle the result of these loaders.
|     },
| };
> export const contactInfo = (args) => <ContactInfo {...args}/>;
| contactInfo.parameters = { storySource: { source: "(args) => <ContactInfo {...args} />" }, ...contactInfo.parameters };
|

**Comment by Rykus0** [comment, marked as answer]
So far I have resolved by removing @storybook/vue, but my project contains both React and Vue components.  It seems I may need to have separate storybook instances for each framework in a subdirectory with its own package.json

**Reply by shilman** [reply]
Yes, we don't really support mixing frameworks. If you want to see multiple frameworks in the same storybook we recommend Storybook Composition, which was built (in part) for this https://storybook.js.org/docs/react/workflows/storybook-composition
